docker/README.md

```diff
@@ -0,0 +1,28 @@
+
+# Running locally via Docker Compose
+
+1. Install Docker: [Docker Desktop](https://docs.docker.com/docker-for-windows/install/) is recommended, but any method that gets Docker and Docker Compose on your machine will do.
+2. Clone Repositories: Clone the other five repositories into the same folder as `compose.yml` (or move `compose.yml` into the folder that contains all of the other repositories into). Your file structure should look something like this:
```

Comment:
cloning git repos inside each other is very poor advice unless you're intending to work with submodules and everyone working on the project knows how to do submodules. they'll be confused about which repo they're making commits to, etc. likewise, advising people to manually copy a file out that requires more manual steps later to keep in sync with updates from git is asking for trouble when half of them forget and are using an outdated file.
better idea, ask devs to clone those repos with the default names _next_ to this repo, and use relative paths in the compose file (e.g. `../project-registry-frontend`)

Reply:
I updated the instructions to have the devs move compose.yml outside the repo rather than the microservices into the repository. I don't *think* versioning on the compose file will be an issue, because the devs shouldn't need to touch it and our group will be hands-off on it by the time the devs start working, but if you think it's better practice I can move the compose to the root of the repository and update accordingly.

Reply:
versioning the compose becomes an issue as soon as the environment variables change, which could happen for any number of reasons.
why does the compose file need to be moved?

Reply:
`compose.yml` and the documentation have been changed to use side-by-side repositories.